plugins/avro-format/core/src/main/java/org/pentaho/di/trans/steps/avro/output/PentahoAvroOutputFormat.java

```diff
@@ -242,4 +251,67 @@ public void setVariableSpace( VariableSpace variableSpace ) {
     this.variableSpace = variableSpace;
   }
 
+  private Object mapDefaultValuesToDataTypes( String defaultValue, AvroSpec.DataType type ) {
```

Comment:
javadoc comment for #mapDefaultValuesToDataTypes

Reply:
addressed

Reply:
addressed